src/module.ts

```diff
@@ -113,7 +113,6 @@ export default defineNuxtModule<ModuleOptions>({
     }
 
     addPlugin({ src: resolve('./runtime/plugins/colors') })
-    addPlugin({ src: resolve('./runtime/plugins/modal') })
     addPlugin({ src: resolve('./runtime/plugins/slideover') })
```

Comment:
I don't see a reason that we need to have this plugins any longer.

Overlays are now managed by the `useOverlayManager` shared composable. We no longer need to `provide/inject` state.